src/UI/Scripts/WorkingHours.js

```diff
@@ -43,7 +43,7 @@ export function areWeOnline(workingHours) {
  * Multiple ranges can be set for the same day.
  * Check the return rules for exact rules used in the check.
  *
- * @param {Array<Array<Number>>} timestamps Format `[[openHoursTimestamp, closeHoursTimestamp],[..., ...]]`
+ * @param {Array<Number>} timestamps Format `[[openHoursTimestamp, closeHoursTimestamp],[..., ...]]`
```

Comment:
Aan de hand van de uitleg lijkt de originele beschrijving eerder te kloppen. Klopt het voorbeeld niet nadat je het type hebt aangepast, of is er een reden waarom er maar 1 array is, terwijl het voorbeeld er 2 lijkt te laten zien?